language/move-prover/tests/sources/stdlib/modules/libra_account.move

```diff
@@ -418,6 +417,12 @@ module LibraAccount {
             fresh_address,
         );
     }
+//    spec fun create_account { // FIXME: This should pass, but Z3 does not terminate on this.
```

Comment:
This is a real issue. This issue needs to be resolved to proceed to the verification of other functions which depend on this function.